File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/arc.py
from Quartz import CGPathCreateMutable, CGPathAddArc, CGPathCloseSubpath, CGPathMoveToPoint, CGPathAddLineToPoint
import math

name = "Arc"

def create_cgpath(frame):
    newpath = CGPathCreateMutable()
    CGPathAddArc(newpath, None, 0,0, math.sqrt(frame.size.width**2 + frame.size.height**2)/2, math.radians(90), 0, True)
    CGPathAddLineToPoint(newpath, None, 0,0)
    return newpath

File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/arc.js
var name = "Arc"


var createPath = function(newPath, frame)
{
    
    newPath.moveToPoint({x:0,y:0});
    newPath.appendBezierPathWithArcWithCenterRadiusStartAngleEndAngleClockwise({x:0,y:0}, Math.sqrt(Math.pow(frame.width,2) + Math.pow(frame.height,2))/2, 0,90, 0);
    
    newPath.closePath();
    
}
